climada/hazard/base.py

```diff
@@ -553,13 +555,13 @@ def from_xarray_raster(
           and Examples) before loading the Dataset as Hazard.
         * Single-valued data for variables ``frequency``. ``event_name``, and
           ``event_date`` will be broadcast to every event.
+        * The ``event`` dimension need not be a time or date.
```

Comment:
I do not understand what this is meaning here.

Reply:
Yes, could be clearer. This way, maybe?
```suggestion
        * The values of the ``event`` coordinate need not be times or dates.
```